tools/test-proxy/Azure.Sdk.Tools.TestProxy/Matchers/CustomDefaultMatcher.cs

```diff
@@ -12,19 +12,28 @@ namespace Azure.Sdk.Tools.TestProxy.Matchers
     public class CustomDefaultMatcher : RecordMatcher
     {
         /// <summary>
-        /// Both compareBodies and nonDefaultHeaderExclusions are safely defaulted for this constructor. This means that providing neither of them will produce a matcher that is functionally identical to the default.
+        /// All optional settings are safely defaulted. This means that providing zero additional configuration will produce a sanitizer that is functionally identical to the default.
         /// </summary>
         /// <param name="compareBodies">Should the body value be compared during lookup operations?</param>
         /// <param name="nonDefaultHeaderExclusions">A comma separated list of additional headers that should be excluded during matching.</param>
+        /// /// <param name="nonDefaultIgnoredHeaders">A comma separated list of additional headers that should be ignored during matching.</param>
         /// <param name="ignoreQueryOrdering">By default, the test-proxy does not sort query params before matching. Setting true will sort query params alphabetically before comparing URI.</param>
-        public CustomDefaultMatcher(bool compareBodies = true, string nonDefaultHeaderExclusions = "", bool ignoreQueryOrdering = false) : base(compareBodies: compareBodies, ignoreQueryOrdering: ignoreQueryOrdering)
+        public CustomDefaultMatcher(bool compareBodies = true, string nonDefaultHeaderExclusions = "", string nonDefaultIgnoredHeaders = "", bool ignoreQueryOrdering = false) : base(compareBodies: compareBodies, ignoreQueryOrdering: ignoreQueryOrdering)
```

Comment:
If it wouldn't cause too much churn, could we rename these to excludedHeaders and ignoredHeaders? I don't think the nonDefault part is really necessary.

Reply:
I honestly thought about this. You're right about making this adjustment now. I'll update the naming and break you in a few places with the name update.